internal/statestore/redis.go

```diff
@@ -71,10 +71,10 @@ func getHealthCheckPool(cfg config.View) *redis.Pool {
 
 	if cfg.IsSet("redis.sentinelHostname") {
 		sentinelAddr := getSentinelAddr(cfg)
-		healthCheckURL = redisURLFromAddr(sentinelAddr, cfg)
+		healthCheckURL = redisURLFromAddr(sentinelAddr, cfg, false)
```

Comment:
Why no password for the healthcheckurl? can you add a note about this to the PR description. The PR description just seems to state this being a test only change but looks like there is more.

Reply:
The redis sentinel library available in golang does not support authentication for now, so I only added the test case for sentinel w/o auth.

Reply:
working on an update